records_mover/db/redshift/redshift_db_driver.py

```diff
@@ -154,3 +156,36 @@ def loader_from_fileobj(self) -> None:
 
     def unloader(self) -> Optional[Unloader]:
         return self._redshift_unloader
+
+    def tweak_records_schema_for_load(self,
+                                      records_schema: RecordsSchema,
+                                      records_format: BaseRecordsFormat) -> RecordsSchema:
+        if isinstance(records_format, AvroRecordsFormat):
```

Comment:
Any thoughts on whether it's appropriate to warn the user here that the types are being downcast? Not sure if that's come up before (or what the exact right mechanism/location for the logic to be), but it might be a helpful hint that "oh hey, your date column is confusingly going to be an integer of days from the epoch". (Might also keep us on our toes for finding ways to improve it in the future!)